### DISCUSSION TITLE
Release draft are not listed in the release page

### DISCUSSION THREAD

**Discussion by hugoalh** [community/community#59535] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

This seems happened starting today, when I wrote a release in my owned repository, click "Save draft," and then navigate back to the repository release page (`https://github.com/user/repo/releases`), the release draft did not appear.

However, if I search my browser history for the release draft URL (`https://github.com/user/repo/releases/edit/untagged-xxxxxxxxxxxxxxxxxxxx`), the release details still exist, apparently the release draft is saved but just not displayed on the repository release page.

*Or it is just happened on me as no one talk about this... :eyes:*

**Comment by ris58h** [comment]
I'm facing the same issue https://github.com/orgs/community/discussions/59533 
It's not only about drafts but normal releases too.

**Reply by hugoalh** [reply]
I can't even saw your discussion on the main page too... Very weird...

The latest one is 1 hour ago :eyes:

![20230628174232](https://github.com/community/community/assets/32359235/328f4c1e-7dd6-40dd-9e08-9b2373479fc0)

**Reply by ris58h** [reply]
@hugoalh it's another bug - I can't see your discussion neither. You should remove all the filters to see it. Seems like newly created discussions aren't searchable.

**Reply by hugoalh** [reply]
Thanks, that's work.

And I found other who facing the same issue:

- https://github.com/orgs/community/discussions/59531
- https://github.com/orgs/community/discussions/59542

**Comment by martylamoureux** [comment]
I just had a draft popping up! One from 2 hours ago

**Reply by ris58h** [reply]
My newly created releases are still invisible.
UPD: they are visible now.

**Reply by martylamoureux** [reply]
I think the queue is getting processed as all of my dummy tests are showing little by little

**Reply by hugoalh** [reply]
Nothing happen on my side :(

**Reply by ris58h** [reply]
@martylamoureux my releases are visible now. The queue assumption seems right.

**Reply by aonez** [reply]
In my case they showed again but now version 1.3.1 shows as the correct release and as an additional draft. Something was messed up.

**Comment by hugoalh** [comment]
Now one of my release draft is visible, exactly 2 hours late 🙄

**Reply by piotrooo** [reply]
I faced with 1 hour delay

**Comment by martin-cb** [comment]
We also have the exact same problem. Releases pop up long time after they are created.

**Comment by kirsan31** [comment]
Same here. Drafts are not visible at all. Latest releases not showing up in releases list. **Also they are absent thru api too.** But on main page I see true latest release (can click on it and edit).
Private repo, latest release is `4.173`, but in the list latest is `4.172` :
![image](https://github.com/community/community/assets/17767561/7944b7ca-5c19-4ffc-9934-11db8397112d)

**Reply by ris58h** [reply]
You have to wait https://github.com/orgs/community/discussions/59535#discussioncomment-6302989

**Comment by hugoalh** [comment]
The issue seems resolved, so it is time to close this.

**Reply by mosioc** [reply]
I am experiencing a problem like this.

**Comment by hrzlgnm** [comment]
Happening again as of today, workaround to get the edit url:
```
gh api repos/OWNER/REPO/releases \
  --jq '.[] | select(.draft == true and .tag_name == "") | .html_url'
```

**Reply by mosioc** [reply]
thanks

**Comment by scomma** [comment]
Still happening. They show up in tags, but not releases.
